pkg/xray/resource/resource_xray_security_policy_test.go

```diff
@@ -1168,3 +1240,30 @@ const securityPolicyMaliciousPkgFixVersionDep = `resource "xray_security_policy"
 		}
 	}
 }`
+
+const securityPolicyPackages = `resource "xray_security_policy" "{{ .resource_name }}" {
+	name = "{{ .policy_name }}"
+	description = "{{ .policy_description }}"
+	type = "security"
+	rule {
+		name = "{{ .rule_name }}"
+		priority = 1
+		criteria {
+			package_name = "{{ .package_name }}"
+			package_type = "{{ .package_type }}"
+			package_versions = ["{{ .package_version_1 }}", "{{ .package_version_2 }}", "{{ .package_version_3 }}"]
+		}
+		actions {
+			block_release_bundle_distribution = {{ .block_release_bundle_distribution }}
+			fail_build = {{ .fail_build }}
+			notify_watch_recipients = {{ .notify_watch_recipients }}
+			notify_deployer = {{ .notify_deployer }}
+			create_ticket_enabled = {{ .create_ticket_enabled }}
+			build_failure_grace_period_in_days = {{ .grace_period_days }}
+			block_download {
+				unscanned = {{ .block_unscanned }}
+				active = {{ .block_active }}
+			}
+		}
+	}
+}`
```

Comment:
I think the tests for invalid attributes combos are missing, i.e. logic from the custom diff func.